docs/config.rst

```diff
@@ -43,13 +43,13 @@ An environment variable like ``REACT_APP_FOO`` controls the setting ``FOO``.
    The audience for the access token that is generated by Auth0.
 
 
-.. envvar:: REACT_APP_CHECK_AUTH_EXPIRY_INTERVAL_SECONDS
```

Comment:
:-1: I think that this setting should stay in seconds, and the code should be fixed to handle the different units. We really don't need the additional precision of milliseconds. It is very difficult to read 300000 and be sure how many zeroes are there.